cmd/shp/main.go

```diff
@@ -4,11 +4,22 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/spf13/pflag"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
 	"github.com/shipwright-io/cli/pkg/shp/cmd"
 )
 
+// ApplicationName application name.
+const ApplicationName = "kubectl-shp"
```

Comment:
We have two application names going on, namely `shp` and when using as a `kubectl` plugin, `kubectl-shp`. Thus, might not be something for this PR, but we will need to find a way to acommodate both.

Reply:
I wil leave it as `shp` for now for a standalone CLI application.